### PR TITLE
tree: Remove some unneeded internal tags in tree

### PR DESCRIPTION
**Pull request by CraigMacomber**
## Description

Remove some unneeded internal tags in tree.

There are still some more, but this gets most of them. I don't have a good tool for finding these, so I didn't get every last one.

## Reviewer Guidance

The review process is outlined on [this wiki page](https://github.com/microsoft/FluidFramework/wiki/PR-Guidelines#guidelines).

### CODE REVIEW

File: packages/dds/tree/src/test/shared-tree/treeNodeApi.spec.ts [hunk 1]
```diff
@@ -47,7 +47,7 @@ describe("treeApi", () => {
 		/**
 		 * Runs a set of transaction tests, either passing the TreeView or the root node to the `runTransaction` function depending on the configuration.
 		 * @remarks This allows for code coverage of both of those variants of the `runTransaction` API without duplicating these tests entirely.
-		 * */
+		 */
```
Comment (CraigMacomber) [inline packages/dds/tree/src/test/shared-tree/treeNodeApi.spec.ts:50]: I was bulk fixing issues like this in comments from the tag removal. I so no reason to revert this unrelated cleanup.

**Review by taylorsw04 — APPROVED** [review]
